pokemongo_bot/cell_workers/move_to_map_pokemon.py

```diff
@@ -341,16 +341,12 @@ def work(self):
                         self.snipe(pokemon)
                         count = count +1
                         if count >= self.config.get('snipe_max_in_chain', 2):
-                            return WorkerResult.SUCCESS
-                        if count is not 1:
-                            time.sleep(self.config.get('snipe_sleep_sec', 2)*5)
```

Comment:
Do you mean the sleep is a must? There already sleeps in snipe function. What does this sleep for?